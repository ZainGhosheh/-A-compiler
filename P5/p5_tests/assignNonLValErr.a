f : () -> int {
    b : immutable int = 3;
	b = 4;
}
