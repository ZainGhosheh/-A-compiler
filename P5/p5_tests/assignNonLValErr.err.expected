FATAL [3,2]-[3,7]: Non-lval assignment
Type Analysis Failed
